common/lib/xmodule/xmodule/css/capa/display.scss

```
@@ -782,6 +782,8 @@ div.problem {
       border-radius: 3px;
       min-width: 160px;
       height: 46px;
+      direction: ltr;
+      text-align: unset;
```

I _think_ this change affects other unrelated input types right?

Please make this change just for the `formulaequationinput` not for other `textline` which may contain RTL text such words or full scentences.

```suggestion
      .inputtype.formulaequationinput input {
          direction: ltr;
          text-align: left;
      }
```

---

Thank you @OmarIthawi , you think right. I've changed it to something more accurate